src/components/routeComponents/memberInformationGrid/SelectedMembersActionWrapper.jsx

```diff
@@ -149,33 +224,60 @@ class SelectedMembersActionWrapper extends Component {
 					borderStyle="none"
 					backgroundColor="unset"
 					padding="0px">
-					<CSVExportButton
-						acl={isCSVExportEnable}
-						selectedMembers={selectedMembers}
-						constants={constants}
-						metaData={metaData}
-					/>
-					<IdCardPrintButton
-						acl={isIdCardPrintEnable}
-						constants={constants}
-						selectedMembers={selectedMembers}
-						checkCoordinators={this.checkCoordinators}
-					/>
-					<UpdateIdCardStatusMembersModal
-						selectedMembers={selectedMembers}
-						updateIdCardStatusModalFormSchema={updateIdCardStatusModalFormSchema}
-						acl={isUpdateIdCardStatusEnable}
-					/>
-					<MarkSelectedMembersAttendance
-						selectedMembers={selectedMembers}
-						attendanceModalFormSchema={attendanceModalFormSchema}
-						acl={isUpdateAttendanceEnable}
-						refreshMembersGrid={refreshMembersGrid}
-						clearSelectedMembers={clearSelectedMembers}
-					/>
+					<StyledButtonRow>
+						<CSVExportButton
+							acl={isCSVExportEnable}
+							selectedMembers={selectedMembers}
+							constants={constants}
+							metaData={metaData}
+						/>
+						<StyledDropdownContainer
+							isEmpty={isEmpty(selectedMembers) ? 'true' : 'false'}>
```

Comment:
No need to send booleans in strings.
```suggestion
							isEmpty={isEmpty(selectedMembers) ? true : false}>
```

Reply:
resolved